### ISSUE TITLE
[1.16.5] Crash with Dotcoin mod upon starting server due to "language provider".

### ISSUE THREAD

**Issue by Darkmega18** [Teneted/Tenet#1711]
**Minecraft Version :** 1.16.5

**Mohist Version :** mohist-1.16.5-764-server

**Operating System :** Windows 10

**Concerned mod / plugin** : https://www.curseforge.com/minecraft/mc-mods/dotcoin-mod (latest 1.16.5 version available.)

**Logs :** https://pastebin.com/s0EUwi82

**Steps to Reproduce :**
Load server with it. 
instant crash before it seems to get to anything else.
There are a lot other mods, but it doesn't seem to get anyway cause this blows up so immediately.

**Description of issue :** Thats kinda it. Just blows up cause of something about a language provider.

**Comment by MohistBOT** [comment]
**Hello, thanks for opening this issue!**

Before continuing, please understand that Mohist is a **free** and **open source** software, made by people in **their free time**. Know that we are not able to give any eta about when your issue will get answered or fixed.

> Our team will first check if your issue isn't duplicated. If it's the case, we will close your issue, giving your the number of the other issue so that you can follow its progress. If you are able to do it, please check that your issue isn't duplicated, this would help us a lot.

*Thank you for your patience, and have a nice day (or night)!*

**Comment by Darkmega18** [comment]
Neat bot. As a side note, I downloaded the java 16 jdk that was available when going through the installation. dunno if that has anything to do with it. was basically freshly installed when I ran it with the run.bat. 

runs perfectly fine as a vanilla thing before I threw my modpack at it of course. and I can't exactly remove the dotcoin mod to see if any others cause issues because I use the coins in a number of datapack based edits like loot tables, villager trades etc.
